### PR TITLE
feat: refresh balance

### PR DESCRIPTION
**Pull request by CarmitKl**
Add two refresh buttons: 1 at the Transfer menu- to refresh the balance of the selected token.
Another in the SelectToken menu- to refresh the balance of all tokens.

### Checklist

- [ ] Changes have been done against master branch, and PR does not conflict
- [ ] New unit / functional tests have been added (whenever applicable)
- [ ] Test are passing in local environment
- [ ] Docs have been updated
- [ ] PR title is follow the [standard-version](https://www.conventionalcommits.org/) convention: `<type>(optional subject): <description>`, e.g: `fix: minor typos in code`

---
This change is [Reviewable](https://reviewable.io/reviews/starkware-libs/starkgate-frontend/116)

### CODE REVIEW

File: src/components/Features/SelectToken/SelectToken.js [hunk 1]
```diff
@@ -30,16 +31,23 @@ export const SelectToken = () => {
...
-        <SearchToken
-          tokens={tokens}
-          onSearchResults={searchResult => setSearchTokens(searchResult)}
-        />
+        <div className="row">
+          <SearchToken
+            tokens={tokens}
+            onSearchResults={searchResult => setSearchTokens(searchResult)}
+          />
+          <RefreshButton onClick={onRefreshClick} />
```
Comment (dan-ziv) [inline src/components/Features/SelectToken/SelectToken.js:49]: Should be under the search bar

File: src/components/UI/NetworkMenu/NetworkMenu.js [hunk 2]
```diff
@@ -2,18 +2,20 @@ import PropTypes from 'prop-types';
...
 
-export const NetworkMenu = ({networkData, tokenData, isTarget, children}) => {
+export const NetworkMenu = ({networkData, tokenData, isTarget, onRefresh, children}) => {
   return (
     <div className={styles.networkMenu}>
       <Badge text={isTarget ? TO : FROM} />
       <div className={styles.networkContainer}>
         <NetworkTitle networkData={networkData} />
         <TokenBalance tokenData={tokenData} />
+        <RefreshButton onClick={onRefresh} />
```
Comment (dan-ziv) [inline src/components/UI/NetworkMenu/NetworkMenu.js:18]: Should be align with the symbol balance and under the `Available balance` text.

**Review by dan-ziv — CHANGES_REQUESTED** [review]
@CarmitKl I see the next following issues:
1. No loading spinner appears when clicking on refresh. So it looks like there's no reaction from the operation.
2. Wrong values for tokens in the Select Token menu.

File: src/components/Features/SelectToken/SelectToken.js [hunk 3]
```diff
@@ -30,16 +31,23 @@ export const SelectToken = () => {
     showTransferMenu();
   };
 
+  const onRefreshClick = () => {
```
[the remark this answers is not in the document]
Comment (CarmitKl) [reply src/components/Features/SelectToken/SelectToken.js]: done

File: src/components/Features/SelectToken/SelectToken.js (hunk 1 above)
Comment (CarmitKl) [reply src/components/Features/SelectToken/SelectToken.js]: done

**Pull request merged**